cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
```

Comment:
UIKit have to be imported
```suggestion
import UIKit
```